app/tests/spec/views/settings/recovery_codes.js

```diff
@@ -82,6 +83,12 @@ describe('views/settings/recovery_codes', () => {
       verified: true
     });
 
+    sinon.stub(account, 'checkTotpTokenExists').callsFake(() => {
```

Comment:
We will want to test two scenarios. 

1. `navigate` gets called with `settings/two_step_authentication` when checkTotpTokenExists is true
2. `navigate` does not get called when checkTotpTokenExists is false